packages/lesswrong/server/emailComponents/EmailWrapper.jsx

```diff
@@ -28,16 +28,17 @@ const EmailWrapper = ({user, unsubscribeAllLink, children, classes}) => {
               {/* 600px container (white background) */}
               <table border="0" width="600" cellPadding="0" cellSpacing="0" className="container" bgcolor="#ffffff">
                 <tr>
-                  <td className="main-container container-padding" bgcolor="#ffffff">
+                  <td className="container-padding" bgcolor="#ffffff">
                     <br/>
                     {children}
                   </td>
                 </tr>
-                <tr><td>
+                <tr><td className="container-padding">
                   <br/><br/>
-                  <a href={unsubscribeAllLink}>Unsubscribe</a>
+                  <a href={unsubscribeAllLink}>Unsubscribe</a>{' '}
```

Comment:
What is that `{'  '}` doing here?

Reply:
It introduces a literal space. That's our standard way of doing so when the html is wrapping but the text is not. You need some way to do so, because the newline is not treated as meaningful by html parsers, which leads to:
> Unsubscribe(from all ...

Reply:
In JSX, whitespace around a line boundary gets stripped, so this previously read `Unsubscribe(from all ...` with a missing space.